index.js

```diff
@@ -53,17 +54,18 @@ const hook = (type, opts, transform) => {
 };
 
 module.exports = (opts, transform) => {
-	const stdoutPromise = hook('stdout', opts, transform);
-	const stderrPromise = hook('stderr', opts, transform);
+	const streams = opts.streams ? arrify(opts.streams) : [process.stdout, process.stderr];
```

Comment:
I think it should only allow an array. The name is already plural and the default is multiple streams. Let's keep the type strict.